src/node/interfaces.cpp

```diff
@@ -88,30 +88,75 @@ class EVOImpl : public EVO
     {
         const CBlockIndex *tip = WITH_LOCK(::cs_main, return ::ChainActive().Tip());
         CDeterministicMNList mnList{};
-        if  (tip != nullptr && deterministicMNManager != nullptr) {
-            mnList = deterministicMNManager->GetListForBlock(tip);
+        if (tip != nullptr && m_context->dmnman != nullptr) {
```

Comment:
```suggestion
        if (tip != nullptr && m_context != nullptr && m_context->dmnman != nullptr) {
```
same for all similar cases

Reply:
Created a `context()` in the style of `chainman()` and used it for all `m_context` access.

Reply:
not sure that is correct solution, should we really call `Assert` here?

if `dmnman` is nullptr - we return failover to `0`. But in m_context is nullptr - why that's an assert? should not change this behaviour maybe?